src/seismometer/table/analytics_table.py

```diff
@@ -532,43 +560,43 @@ def __init__(
             options=[THRESHOLD] + MONOTONIC_METRICS,
             value=metric,
             description="Metric",
-            style=WIDE_LABEL_STYLE,
+            style={"description_width": "40px"},
+            layout=Layout(width="calc(max(max-content, var(--jp-widgets-inline-width-short)))", min_width="200px"),
         )
         self._metrics_to_display = MultiselectDropdownWidget(
             options=[THRESHOLD] + STATNAMES + OVERALL_PERFORMANCE,
             value=metrics_to_display or GENERATED_COLUMNS,
             title="Performance Metrics to Display",
         )
-        self._metric_values_slider = FloatRangeSlider(
-            min=0.01,
-            max=1.00,
-            step=0.01,
-            value=metric_values or [0.2, 0.8],
-            description="Metric Values",
-            style=WIDE_LABEL_STYLE,
+        metric_values = metric_values or (0.8, 0.2)
+        self._metric_values = MonotonicProbabilitySliderListWidget(
+            names=("Metric Value 1", "Metric Value 2"), value=tuple(metric_values), ascending=False
         )
         self._group_by = Dropdown(
             options=["Score", "Target"],
             value="Score",
             description="Group By",
-            style=WIDE_LABEL_STYLE,
+            style={"description_width": "60px"},
```

Comment:
```suggestion
            style={"description_width": "min-content"},
```

width: min-content fixes the labels, but something is still off with the end of the drop down

Reply:
I think applying `align-items: flex-end;` here might work

Reply:
You probably want to group this with in a VBox with the metric dropdown

Reply:
Tried putting them together in a VBox but since the descriptions have different lengths it does not look as good as the current structure.